cogs/whisper.py

```diff
@@ -23,7 +23,7 @@ async def psst(self, ctx, *,message: str):
         seed = os.environ.get('WHISPERHASH')
         channel = self.bot.get_channel(689655916223660095)
         namehash = abs(hash(ctx.message.author.mention)*hash(seed)) % (10 ** 8)
-        await channel.send(self.random_adjectives[namehash % len(self.random_adjectives)] + " " + self.random_animals[namehash -24 % len(self.random_animals)] + ":  " + message)
+        await channel.send("~~{name}:~~  {msg}".format(name=str(self.random_adjectives[namehash % len(self.random_adjectives)], self.random_animals[namehash -24 % len(self.random_animals)]), msg = message))
```

Comment:
Looking great! That line length is crazy though, but I won't judge.

Reply:
Psst, here's something cool you can use next time: https://realpython.com/python-f-strings/